src/server/bitops_family_test.cc

```diff
@@ -41,7 +200,7 @@ TEST_F(BitOpsFamilyTest, GetBit) {
 
   // make sure that when accessing bit that is not in the range its working and we are
   // getting 0
-  EXPECT_EQ(0, CheckedInt({"getbit", "foo", std::to_string(strlen("abc") + 5)}));
+  // EXPECT_EQ(0, CheckedInt({"getbit", "foo", std::to_string(strlen("abc") + 5)}));
```

Comment:
why commenting out?

Reply:
I will change it to return key not found for the not operation. I will add more comments in the code to cover it

Reply:
this was commented - not sure I it got commented